cmd/cloud_run.go

```diff
@@ -0,0 +1,376 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"go.k6.io/k6/cloudapi"
+	"go.k6.io/k6/cmd/state"
+	"go.k6.io/k6/errext"
+	"go.k6.io/k6/errext/exitcodes"
+	"go.k6.io/k6/lib"
+	"go.k6.io/k6/lib/consts"
+	"go.k6.io/k6/ui/pb"
+)
+
+// cmdCloudRun handles the `k6 cloud` sub-command
+type cmdCloudRun struct {
+	gs *state.GlobalState
+
+	showCloudLogs bool
+	exitOnRunning bool
+	uploadOnly    bool
+}
+
+func getCmdCloudRun(gs *state.GlobalState) *cobra.Command {
+	c := &cmdCloudRun{
+		gs:            gs,
+		showCloudLogs: true,
+		exitOnRunning: false,
+		uploadOnly:    false,
+	}
+
+	exampleText := getExampleText(gs, `
+  # Run a test in the Grafana k6 cloud
+  $ {{.}} cloud run script.js
+
+  # Run a test in the Grafana k6 cloud with a specific token
+  $ {{.}} cloud run  --token <YOUR_API_TOKEN> script.js`[1:])
```

Comment:
```suggestion
  $ {{.}} cloud run --token <YOUR_API_TOKEN> script.js`[1:])
```

Reply:
I don't think the `--token` flag is supported for now, do we want to add support for it? 🤔

Reply:
No, I think it's definetly a mistak on my side, I did both cloud run and cloud login at the same time and I must have updated `cloud login` without realizing I did 👍🏻 

Dropping it 👍🏻